### PR TITLE
Updating env.test.sample

### PR DESCRIPTION
**Pull request by joshuatobin**
CC: @heroku/sre

### CODE REVIEW

**Comment by lexelby** [comment]
:+1:

File: .env.development.sample [hunk 1]
```diff
@@ -0,0 +1,8 @@
+SOURCE_URL="https://status.localhost.com/v4/api/incidents"
```
Comment (jmervine) [inline .env.development.sample]: I would make SOURCE_URL examples more generic given that our status site is not open source...

```
SOURCE_URL=https://status.example.com/api/endpoint
```

Or something like that.

**Comment by joshuatobin** [comment]
@jmervine Agreed. I've updated the `source_url` to be generic.

**Comment by reidmix** [comment]
👍

**Pull request merged**